example/aes.dart

```diff
@@ -0,0 +1,48 @@
+library example.aes;
+
+import "dart:convert";
+import "dart:typed_data";
+
+import "package:hex/hex.dart";
+import "package:pointycastle/pointycastle.dart";
+
+void main() {
+  // Key must be multiple of block size (16 bytes).
+  var key = new Digest("SHA-256").process(
+      UTF8.encode("correct horse battery staple"));
```

Comment:
Use new `utf8` (lowercase). I would also highly recommend to set any non-reassigned values (like this one) as `final` instead, which is generally considered a good practice.

Reply:
Also, since we're now in the Dart 2 age, I would remove all instances of `new`.